mnt/reform2-nitrogen8m/kernel-config

```diff
@@ -0,0 +1,2180 @@
+6LOWPAN n
```

Comment:
Is this original work?

If it isn't, where can we get the updated recommended options to enable?

Reply:
This is semi-original, I took the config of the [stock MNT kernel](https://source.mntmn.com/MNT/reform-system-image/src/branch/reform2-redo/reform2-imx8mq/template-kernel/kernel-config), diffconfig'ed and trimed it a bit. Some stuff is disabled to reduce the compile time. MNT uses their patches against a 5.7 kernel, so I chose the same.